tests/qml/tst_VPNRadioDelegate.qml

```diff
@@ -0,0 +1,71 @@
+import QtQuick 2.15
```

Comment:
It seems we don't set the header in the qml test files. This is wrong, we should. Can you please add:

```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

We can add the same header in the other files as a follow-up.